Perl/Perl.sublime-syntax

```diff
@@ -3,11 +3,12 @@
 # http://www.sublimetext.com/docs/3/syntax.html
 name: Perl
 file_extensions:
+  - pc
```

Comment:
I believe the order of these affects the save dialog filter options on Windows, in which case we'd want `.pl` to come first.